pages_/logout.tsx

```diff
@@ -10,6 +11,13 @@ export default function Logout() {
 }
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
+  const token = getToken(context);
+
+  // Allow to logout only shops
+  if (process.env.NODE_ENV === 'production' && !token?.clientId) {
```

Comment:
`!token?.clientId` shouldn't that be shopId?

Reply:
yes